### ISSUE TITLE
More information 

### ISSUE THREAD

**Issue by okanduzyel** [maccoylton/esp-homekit-multi-sensor#3]
Your projects are very nice but, Please give more information about for your whole projects. 

Such as, for this project, Read me doesn't give these answers of questions at below. 

- Which light sensor did you use?
- Wiring schematics?
- Resistors, Capacitors, Inductor values?
- Any Caution? or Suggestions? or Building instructions?

Thanks. ✋🏻

**Comment by maccoylton** [comment]
- Light Sensor, connected to Analog pin, I used one like this:-  https://www.ebay.co.uk/itm/LDR-Photoresistor-Light-Detection-Sensor-Module-Dependent-Resistor-Arduino-PIC/274289838944?hash=item3fdcf2cb60:g:ArUAAOSwGyleWUjE
- LED on GPIO 13 with a suitable resistor dependant on your LED
- Motion sensor HC-SR501 on GPIO  5
- Temperature sensor DHT22 on GPIO 4

Easiest way to get started it to wire it all up and then flash the firmware OTA. Checkout the Wiki I've added

**Comment by okanduzyel** [comment]
I did it with simple LDR resistor and some voltage divider resistors. As you know, input of esp's adc allows max. 1v. Therefore, I had set up that circuit at below. 

<img width="665" alt="Ekran Resmi 2020-06-24 10 38 11" src="https://user-images.githubusercontent.com/38145537/85516816-4c56ba80-b607-11ea-9af7-92a5372f1e17.png">

It simply worked for me but I don't think it gives truth values of light. Because, It requires to be scaling. I need opamp circuit to mirror its light range to 0-1v range.
